src/osd/PGLog.h

```diff
@@ -926,6 +927,8 @@ struct PGLog : DoutPrefixProvider {
     const bool object_not_in_store =
       !missing.is_missing(hoid) &&
       entries.rbegin()->is_delete();
+    ldpp_dout(dpp, 10) << __func__ << ": hoid " << " object_not_in_store: "
```

Comment:
I think you meant to have to have this here.

`ldpp_dout(dpp, 10) << __func__ << ": hoid " << hoid << " object_not_in_store: "`